docs/release-notes/release-notes-0-18-0.md

```diff
@@ -181,7 +181,7 @@ The following defects were fixed in this release.
 | GUI-1823  | Fixed dashboard typo in CPU Usage Global Metrics; graph showed value in seconds instead of ms.  |
 |   |   |
 |   |   |
-|   |   |
+| STRM-122  | Streams can be created with invalid stream names, such as "123" or "_#".  |
```

Comment:
@diana-macrometa - `STRM-122` disallows creating streams with invalid names. Please correct the note.

Reply:
This is the defect that was fixed. It's a list of things that were wrong, now no longer are.

Reply:
Got it, approved.